util.cc

```diff
@@ -258,6 +258,19 @@ xfopen(const char* fname, const char* type, const char* errtxt)
   return f;
 }
 
+/*
+ * OS-abstracting wrapper for getting Unicode environment variables.
+ */
+QString ugetenv(const char* env_var) {
+#ifdef __WIN32__
+  // Use QString to convert 8-bit env_var argument to wchar_t* for _wgetenv().
```

Comment:
qtenv does us no favors here. Ugh.
http://doc.qt.io/qt-5/qtglobal.html#qgetenv

Reply:
Yup. Supposedly in Qt 5.10 there's finally a Unicode-supporting qgetenv or somesuch, but not until then.